src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import ball.characteristics.Ball;
+import java.util.Random;
+
+public class Lottery {
+    public static String getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(ColorSupplier.getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
+        return "Ball color is " + ball.getColor() + " and the number is " + ball.getNumber();
+    }
```

Comment:
Don't use static methods in your solution. Instead, create an instance of the class whose method you want to call. This will make your code more flexible and object-oriented.

Reply:
nice hint